### PR TITLE
Clean up issue templates

### PR DESCRIPTION
**Pull request by bgilbert**
:bike: :house:

The resulting chooser looks like [this](https://github.com/bgilbert/fedora-coreos-tracker/issues/new/choose).

cc @miabbott @dustymabe

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -19,12 +19,15 @@ Steps to reproduce the behavior:
...
+A clear and concise description of what actually happened.
+
+**System details**
+ - Bare Metal/QEMU/AWS/GCP/etc.
  - Fedora CoreOS version
 
-**Ignition Configuration**
-Please attach your FCCT or Ignition configuration used to provision your system. If not using FCCT to generate your Ignition config, does the Ignition config pass validation using [ignition-validate](https://coreos.github.io/ignition/getting-started/#config-validation)?
+**Ignition config**
+Please attach your FCC or Ignition config used to provision your system. If not using FCCT to generate your Ignition config, does the Ignition config pass validation using [ignition-validate](https://coreos.github.io/ignition/getting-started/#config-validation)?
```
Comment (arithx) [inline .github/ISSUE_TEMPLATE/bug-report.md:30]: Worth a subtle nod to cleaning out any potential secrets?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <!--
-If reporting a bug in Fedore CoreOS, please include the
+If reporting a bug in Fedora CoreOS, please include the
```
Comment (arithx) [inline .github/ISSUE_TEMPLATE.md:2]: 😅

**Review by dustymabe — APPROVED** [review]
Works for me

File: .github/ISSUE_TEMPLATE/new-package.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 name: Request a new package
 about: Ask for a new package to be added to Fedora CoreOS
 title: 'New Package Request: <package name>'
-labels: ''
+labels: 'kind/enhancement'
```
Comment (jlebon) [inline .github/ISSUE_TEMPLATE/new-package.md:5]: On that note, maybe we should have a new label for new package requests. Not blocking of course.

**Review by jlebon — APPROVED** [review]
LGTM!

**Review by miabbott — APPROVED** [review]
~~Fix that one spelling item that slowrie found and let's merge.~~  (nm, can't read this morning :coffee: )

Thanks for the cleanup work, Benjamin!

**Comment by miabbott** [comment]
Ah dang, squashed the commits.  :(

**Pull request merged**